src/main/resources/assets/tconstruct/book/materials_and_you/sections/tools.json

```diff
@@ -1,32 +1,37 @@
 [
   {
-    "name": "repairing",
+	"name": "terminology",
     "type": "text",
+    "data": "tools/terminology.json"
+  },
+  {
+	"name": "repairing",
```

Comment:
Its a bit weird that we tell you how to repair tools before we tell you how to make tools. Probably better to put this at the end of the section